### ISSUE TITLE
Add action on click

### ISSUE THREAD

**Issue by SyedSaal** [kevinsqi/react-circular-progressbar#100]
### Add action on click

Hey, I'm developer timer progress bar and I want to give an action after the completion of set timer 

### Suggested solution (optional)

_Suggest a solution that would solve the problem you have. For example, this may involve adding a new prop, changing an existing prop, or adding a new exported component._

**Comment by kevinsqi** [comment]
Yeah, this makes sense to me. I can take a look at some point, but PRs are welcome.

**Comment by navarro-david** [comment]
I would like to add on to this feature request. In the scenario where we have multiple paths in a single circular progress bar, it would be great to add an action to each of those paths.

**Comment by SyedSaal** [comment]
I added an action.. below is my code

`const timerView = !isAlreadyVisitedItem && (
      <div onClick={() => this.nextItemClick(true)}>
        <CircularProgress>
          <ChangingProgressProvider value={progressCurrentValue}>
            {percentage => (
              <CircularProgressbar
                value={percentage}
                minValue={0}
                maxValue={30}
                styles={buildStyles({
                  pathTransitionDuration: 1,
                  pathColor: "white",//turquoise
                  //trailColor: "gold",
                })} />
            )}
          </ChangingProgressProvider>
        </CircularProgress>
      </div>
    );`

**Comment by wazcov** [comment]
+1 for adding on click

**Comment by hamed-farag** [comment]
+1 for adding on click

**Comment by taejs** [comment]
+ 1

**Comment by phdog** [comment]
+1
